src/doc/unstable-book/src/language-features/lang-items.md

```diff
@@ -23,7 +23,9 @@ use core::panic::PanicInfo;
 
```

Comment:
The text above should also be changed to not mention two lang items (line 14) (Github won't let me add a comment there)